faq.md

```diff
@@ -20,5 +20,5 @@ Yes, after a few months we finally found the answer. Sadly, Mike is on vacations
 
 Versions of resource and infrastructure modules should be specified. Providers should be configured outside of modules, but only in composition. Version of providers and Terraform can be locked also.
 
-There is no master dependency management tool, but there are some tips to make dependency hell less problematic. For example, [Dependabot](https://dependabot.com/) can be used to automate dependency updates. Dependabot creates pull requests to keep your dependencies secure and up-to-date. Dependabot supports Terraform configurations.
+There is no master dependency management tool, but there are some tips to make dependency specifications less problematic. For example, [Dependabot](https://dependabot.com/) can be used to automate dependency updates. Dependabot creates pull requests to keep your dependencies secure and up-to-date. Dependabot supports Terraform configurations.
```

Comment:
"Dependency hell" is a valid term in this context, too. https://en.wikipedia.org/wiki/Dependency_hell